src/CardsAPI/Card.py

```diff
@@ -79,13 +81,13 @@ def value(self, arg):
             self.name = value_to_card_dict[arg]
         self._value = card_to_value_dict[self.name]
 
-    def __radd__(self, other: Union['Card', int]) -> int:
+    def __radd__(self, other: Union["Card", int]) -> int:
```

Comment:
Is there a rule for the use of  `'` and `"` ? Or it's just your convenient ?

Reply:
https://www.python.org/dev/peps/pep-0008/#string-quotes
TLDR: No rule, but try to be consistent within the same file/project. I personally prefer double quotes because you can safely include apostrophes in the string

Reply:
It's what I though. I used simple quotes for simple words, and use double quotes for sentences. 
Do you think that we can have apostrophes in our class names ?

Reply:
In class names we can't have apostrophes, definitely. Actually I wouldn't have changed those to double quotes myself, Black did it when running, I don't mind but if you prefer single quotes I can change it back

Reply:
For me, it's more consistent to use simple quotes, as used before ;)

Reply:
Fixed